### ISSUE TITLE
[FEATURE] Adding .nvmrc and matching OSD node version

### ISSUE THREAD

**Issue by amitgalitz** [opensearch-project/opensearch-dashboards-functional-test#400]
### Is your feature request related to a problem?
We currently don't have a way to enforce what node version is being used, I believe it makes sense to match whatever OSD is using. We should add a `.nvmrc` file at the very least manually matching what is in OSD.

### What solution would you like?
Adding `.nvmrc` file, doc for using `nvm use` and a test checking version match same as OSD has. 

Just opening an issue incase there is something I'm not aware of that means test repo shouldn't have same node version as OSD. If not then I'll quickly add this.

**Comment by ashwin-pc** [comment]
I like this, do you want to make a quick contribution for this @amitgalitz?

**Comment by tianleh** [comment]
It is using the node in the Docker image for running test

**Comment by ashwin-pc** [comment]
Yep i ran into this too, and using a different version of node actually broke local testing for me.
